universitas-negeri-semarang-fakultas-matematika-dan-ilmu-pengetahuan-alam.csl

```diff
@@ -0,0 +1,427 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" and="symbol" et-al-min="1" demote-non-dropping-particle="sort-only" default-locale="id-ID">
+  <info>
+    <title>Universitas Negeri Semarang-Fakultas Matematika dan Ilmu Pengetahuan Alam</title>
+    <title-short>UNNES-FMIPA</title-short>
+    <id>http://www.zotero.org/styles/universitas-negeri-semarang-fakultas-matematika-dan-ilmu-pengetahuan-alam</id>
+    <link href="http://www.zotero.org/styles/universitas-negeri-semarang-fakultas-matematika-dan-ilmu-pengetahuan-alam" rel="self"/>
+    <link href="http://fisika.unnes.ac.id/wp-content/uploads/2010/03/Panduan-Penulisan-Skripsi-TA-Artikel-rev-2014.pdf" rel="documentation"/>
```

Comment:
You have changed the "self" link to a "template" link, but we need both.

The "self" link should remain `<link href="http://www.zotero.org/styles/universitas-negeri-semarang-fakultas-matematika-dan-ilmu-pengetahuan-alam" rel="self"/>`, and the "template" link should give the link to the CSL style you based this one on (unless you wrote it from scratch, in which case we don't need it).